app/forms/state_file/archived_intakes/verification_code_form.rb

```diff
@@ -0,0 +1,24 @@
+module StateFile
+  module ArchivedIntakes
+    class VerificationCodeForm < Form
+      attr_accessor :verification_code, :email_address
```

Comment:
sorry i should have caught this during the mob review but why `attr_accessor` instead of `set_attributes_for`? is it because we're not actually updating any db records?